Tasks/PublishSymbols/PublishSymbols.ps1

```diff
@@ -20,7 +20,7 @@ import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 
 # Warn if deprecated parameter was used.
-if ($sourceFolder -ne $env:Build_SourcesDirectory) {
+if ($sourceFolder -and ($sourceFolder -ne $env:Build_SourcesDirectory)) {
```

Comment:
Fix for definitions edited/created after m88. They should not print the warning and the variable will be empty in those definitions. Whereas pre-88 the default value was Build.sourcedir